praw/objects.py

```diff
@@ -640,15 +669,113 @@ def _update_submission(self, submission):
                 reply._update_submission(submission)  # pylint: disable=W0212
 
     @property
+    def default_contextlink(self):
```

Comment:
Can you please instances of `contextlink` with `context_link`? In fact, maybe `context_url` is better?